model/task_lifecycle.go

```diff
@@ -865,7 +865,7 @@ func getBuildStatus(buildTasks []task.Task) string {
 	// not started
```

Comment:
Can we tweak this comment since it's not clear cut anymore? Maybe a full sentence

Reply:
fixed